starter-code/src/App.js

```diff
@@ -1,21 +1,234 @@
 import React, { Component } from 'react';
 import logo from './logo.svg';
 import './App.css';
+import foods from './foods.json'
+
+class FoodTag extends Component {
+  constructor(props) {
+    super(props)
+    this.state = { qty: 1 }
+  }
+
+  handleChange(event) {
+    this.setState({ qty: parseInt(event.target.value) })
```

Comment:
It is a good practice to pass radix with parseInt like so:
```js
parseInt(string, radix)
```
Also, you have an alternative to parseInt(), which is Number()